### ISSUE TITLE
fastfood should provide a command for automatically syncing file changes with manifest file listings

### ISSUE THREAD

**Issue by samstav** [rackerlabs/fastfood#20]
For example, if I add or remove files in `templatepack/stencils/redis/`, I should be able to run a simple `fastfood` command that will sync up the changes I've made in the stencil with the `'files'` property in the manifest file ( `stencils/redis/manifest.json` ). 

It could add (or remove from) the `default_stencil`'s `'files'` section in the manifest by default, or you could specify the stencil(s) it applies to, e.g. ( master, slave, sentinel or all)

**Comment by samstav** [comment]
@jarosser06 @mattjbarlow Does this sound right?

**Comment by jarosser06** [comment]
Im not sure I follow, so fastfood would be able to manipulate the template pack?  If you make changes to the stencil set and want to populate those changes into a cookbook that had a previous version you could just run fastfood gen with the -f flag.

**Comment by samstav** [comment]
Yes this would mean fastfood would be able to manipulate the templatepack. My suggestion was that it would be able to update the `'files'` property in the manifest.json for a stencil when it saw files added to the stencil somewhere that weren't yet listed in the manifest.json. ( it should also be able to remove files from the manifest.json if it sees they were removed)

**Comment by jarosser06** [comment]
Ahh I think I understand so if I add a new template to the stancil set then fastfood can find it and update my manifest.json file?  The only concern is that because templates are per stencil and a stencil set can have multiple stencils how would fastfood know what stencil that file should belong too.

It may be helpful for fastfood to be able to run a check on a templatepack tho, maybe look for orphaned templates that do not exist in any manifest and report them or report any templates listed in a manifest that don't actually exist.

**Comment by jarosser06** [comment]
Note to self: Fastfood should be able to determine what files were created with fastfood and print warnings to let an end user know what has deviated from a template pack.
